migrations/schema.sql

```diff
@@ -1365,10 +1368,8 @@ UNION
  SELECT NULL::bigint AS post_id,
     u.id AS user_id,
     NULL::bigint AS comment_id,
-    ((setweight(to_tsvector('english'::regconfig, (u.name)::text), 'A'::"char") || setweight(to_tsvector('english'::regconfig, COALESCE(string_agg(replace((t.name)::text, '-'::text, ' '::text), ' '::text), ''::text)), 'C'::"char")) || setweight(to_tsvector('english'::regconfig, COALESCE(u.bio, ''::text)), 'C'::"char")) AS document
```

Comment:
yes. it means the version of the search_index materialized view in your local db doesn't match the current one. you can always `FullTextSearch.dropView().then(() => FullTextSearch.createView())` to bring it up to date.